ed_navigation/src/navigation_plugin.cpp

```diff
@@ -14,6 +14,7 @@
 #include <ros/console.h>
 
 #include <iomanip>
+#include <memory>
```

Comment:
I don't think memory is used in this file

Reply:
It is needed for `std::const_pointer_cast`, https://en.cppreference.com/w/cpp/memory/shared_ptr/pointer_cast